### DISCUSSION TITLE
Error doing POST in Create

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#6519] [Help] [answered]
I'm making a post to record records in the BD but it's giving an error in SAVEREQUEST
Can anyone help.
![image](https://user-images.githubusercontent.com/7516443/186971381-fdd18f68-be2c-4826-b53a-06e9cce20bda.png)

**Comment by vooba** [comment, marked as answer]
SaveRequest Format:
```ts
export interface SaveRequest<TEntity> extends ServiceRequest {
    EntityId?: any;
    Entity?: TEntity;
    Localizations?: any;
}
```

Try to use:
```js
{
    Entity: { 
       Nome: "test",
      /* your entity */" 
    }
}
```

**Reply by juliermefelix** [reply]
Thank you very much, it worked.
